src/elm/Session/Guest.elm

```diff
@@ -151,15 +202,38 @@ viewLeftCol mdWidth =
 
 viewPageHeader : Model -> Shared -> Html Msg
 viewPageHeader model shared =
-    header
-        [ class "flex items-center justify-between pl-4 md:pl-6 py-3 bg-white" ]
-        [ a [ Route.href (Route.Login Nothing) ]
-            [ img
+    let
+        imageElement url =
+            img
                 [ class "h-5"
-                , src shared.logo
-                , alt "Cambiatus"
+                , src url
                 ]
                 []
+
+        loadingSpinner =
+            div [ class "full-spinner-container h-full" ]
+                [ div [ class "spinner spinner--delay" ] [] ]
+
+        logo =
+            case model.community of
+                Loading ->
+                    ""
+
+                Loaded comm ->
+                    comm.logo
+
+                Default ->
+                    shared.logo
+    in
+    header
+        [ class "flex items-center justify-between pl-4 md:pl-6 py-3 bg-white" ]
+        [ a [ Route.href (Route.Login Nothing) ]
+            [ case model.community of
+                Loading ->
+                    loadingSpinner
+
+                _ ->
+                    imageElement logo
```

Comment:
I think we could be a bit trickier here.

Now only the default Cambiatus community is shown as [the design](https://www.figma.com/file/47BEsGEMSWfhTm3zC0ugyA/Ticolones-version?node-id=3%3A2206) suggests:
![image](https://user-images.githubusercontent.com/140053/93628824-df797380-f9ef-11ea-973d-43bc8c47837a.png)

Since the Ticolones is an important community, we have to make sure that it looks according to the branding. We could have an ad-hoc `if` statement for the `TICO` symbol and show the SVG logo stored in the repo:
![image](https://user-images.githubusercontent.com/140053/93629139-780ff380-f9f0-11ea-9b33-9950fe1ac4cf.png)

For all other communities, it would be great to show a title next to the logo:
![image](https://user-images.githubusercontent.com/140053/93629226-98d84900-f9f0-11ea-99f1-229ec4309c73.png)

I tried this code and it worked pretty fine:

```elm
Loaded c ->
    span [ class "whitespace-no-wrap" ]
        [ imageElement logo
        , span [ class "inline-block align-middle ml-1 font-bold capitalize" ]
            [ text c.title ]
        ]
```

Reply:
I talked to Helton about this on Slack, and we came to the conclusion that it's best to just show the community logo, as that lets community owner upload logos with their own fonts and styles if they wish to do so.

Reply:
Ok, that seems reasonable. But what about cases like this where we show only the sign (icon) without the text part?
![image](https://user-images.githubusercontent.com/140053/93631708-ba3b3400-f9f4-11ea-9371-1001441262b1.png)

Reply:
maybe we could have two logos, with default sizings. But for now we may be inconsistent

Reply:
According to him that's not necessarily an issue, and we should just let users know of what the logo should contain. @heltonlr can you explain it here please?

Reply:
I said that we cannot use text for the logo, the correct is to use an image.  If the "ICON + TEXT"a logo gets bad due to the platform limitation, we can only use the "ICON"s logo (as we did with Muda).